bambam.py

```diff
@@ -107,12 +113,20 @@ def input(events, quit_pos):
                         sound_muted = True
                         pygame.mixer.fadeout(1000)
                         sequence = ''
-            
+
             # Clear the background 10% of the time
             if random.randint(0, 10) == 1:
                 screen.blit(background, (0, 0))
                 pygame.display.flip()
-            
+
+            # send command to yeelight
+            if yee and args.yee:
+                if args.deterministic_sounds:
+                    color = event.key * 0xedcba % 0x1000000
```

Comment:
Please explain these magic values.

Reply:
The idea was to just evenly spread values across spectrum. `0x1000000` is (white + 1), and `0xedcba` is just an arbitrary value to make sure resulting values won't be within the same small range.